sql/mysql-compatibility.md

```diff
@@ -140,6 +140,10 @@ Create Table: CREATE TABLE `t1` (
 
 Architecturally, TiDB does support a similar storage engine abstraction to MySQL, and user tables are created in the engine specified by the [`--store`](server-command-option.md#--store) option used when you start tidb-server (typically `tikv`).
 
+### Partitioning
+
+TiDB supports only [`RANGE` Partitioning](https://dev.mysql.com/doc/refman/5.7/en/partitioning-range.html). Support for `LIST`, `HASH`, `KEY` and subpartitioning is not available at this time.
```

Comment:
We should mention that partitioning is not enabled by default. See https://github.com/pingcap/tidb/blob/master/sessionctx/variable/tidb_vars.go#L110
/cc @tiancaiamao

Reply:
I believe we can enable range partition by default on the master branch within this month.